### PR TITLE
Create  List structure 

### PR DESCRIPTION
**Pull request by Errytagedesign**
## The following was added to this repository:

- Add list structure
- Dynamically render list with Javascript
- Add webpack boilerplate
- Add linters

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Errytagedesign, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has your portfolio data on the readme, kindly change them to this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -110,7 +90,8 @@ Run this command:
 ### Usage
```
Comment (Efiamotu-1) [inline README.md:90]: - [ ] Your setup as shown below points the user to your microverse portfolio project rather it should point the user to your todolist project. I kindly suggest you make this change to your readme file 👍 
- [ ] Also your install section should be corrected as well
setup

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-plusplus */
+import './index.css';
+
+const displayList = document.querySelector('.todoList');
+
+const toDoList = [
+  { description: 'Read About Express.js', completed: false, index: 0 },
+  { description: 'Clean the home office', completed: false, index: 1 },
+  { description: 'Complete 1000 steps walk', completed: true, index: 2 },
+];
```
Comment (Efiamotu-1) [inline src/index.js:10]: - [ ] I want to kindly suggest your index should begin at `1` rather than `0`.

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Errytagedesign, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Great job setting up webpack
- ✔️ Good job implementing the todos dynamically 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frempongdev — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️  

Hi @Errytagedesign, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
all application features work :heavy_check_mark:
all corrections made :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**